client-side/src/components/settings/manageNotifications/EmailFrequency.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import Select from '../../../stories/Select/Select.jsx';
+import GenericButton from '../../../stories/Button/GenericButton.jsx';
+
+const emailFrequencyEnum = {
+  'never':'🚫',
+  'daily':'📅',
+  'weekly':'🗓️',
+  'monthly':'📆',
+  'yearly':'📅'
+};
+
+
+const EmailFrequency = () => {
+  const [emailFrequency, setEmailFrequency] = useState('');
```

Comment:
why does emailFrequency needs to be in state? on user clicks on save buuton, you get the value from the select and save it